starport/services/network/queries.go

```diff
@@ -116,3 +128,17 @@ func (n Network) GenesisValidators(ctx context.Context, launchID uint64) (genVal
 
 	return genVals, nil
 }
+
+// ChainReward fetches the chain reward from SPN by launch id
+func (n Network) ChainReward(ctx context.Context, launchID uint64) (rewardtypes.RewardPool, error) {
+	n.ev.Send(events.New(events.StatusOngoing, fmt.Sprintf("Fetching rewards for chain %d", launchID)))
+	res, err := rewardtypes.NewQueryClient(n.cosmos.Context).RewardPool(ctx, &rewardtypes.QueryGetRewardPoolRequest{
+		LaunchID: launchID,
+	})
+	if status.Code(err) == codes.InvalidArgument {
```

Comment:
`codes.InvalidArgument` represents the `not found` error. also, it can be more than one type of error, but these errors are hard to happen in this simple query, but

Reply:
If it's not found, shouldn't we let the user know anyway? Even if there is a possibility, it should be handled I think.

Reply:
If not found, we print the `-` into the chain summary, but if the query returns another error, like timeout, we should handle the error.

Reply:
services/network is some sort of a controller. We're right now using it in the cmd only but it doesn't mean that it won't be used programmatically for backend apps. In that such usage scenario it is better if this func returns a not found error.

it's up to the client, -in our case it's the cmd package- to handle that error by printing it out directly or hiding with the `-` placeholder.